RamilyaNigmatullina/pdp-2
# [#159385490] Add google and facebook oauth

# Summary

As User I should be able to Sign In through Facebook and Google

https://www.pivotaltracker.com/story/show/159385490

# Test plan

* Go to Application
* Click on "Sign In with Google" link
* See the the message "Signed in successfully." and go to the settings page to make sure that the profile was created correctly with right info
* Log out
* Click on "Sign In with Facebook" link
* See the the message "Signed in successfully." and go to the settings page to make sure that the profile was created correctly with right info
